src/components/Patient/EncounterQuestionnaire.tsx

```diff
@@ -48,9 +48,10 @@ export default function EncounterQuestionnaire({
     >
       <div className="flex flex-col space-y-4">
         {encounter && (
-          <EncounterHeader
-            encounter={encounter}
-            canWriteSelectedEncounter={false}
+          <PatientHeader
+            patient={encounter.patient}
+            facilityId={facilityId}
+            className="bg-white shadow-sm border-none rounded-sm"
```

Comment:
is border-none really required? 🤔 

```suggestion
            className="bg-white shadow-sm rounded-sm"
```